hadoop-ozone/cli-admin/src/main/java/org/apache/hadoop/hdds/scm/cli/ContainerOperationClient.java

```diff
@@ -221,6 +221,33 @@ public ContainerWithPipeline createContainer(HddsProtos.ReplicationType type,
     }
   }
 
+  @Override
+  public ContainerWithPipeline createContainer(ReplicationConfig replicationConfig,
+                                                String owner) throws IOException {
+    XceiverClientSpi client = null;
+    XceiverClientManager clientManager = getXceiverClientManager();
+    try {
+      if (replicationConfig == null) {
+        replicationConfig = ReplicationConfig.fromProtoTypeAndFactor(replicationType, replicationFactor);
+      }
```

Comment:
Use `Objects.requireNonNull` to verify `replicationConfig` and avoid fallback to type/factor from config.